# SPDX-FileCopyrightText: 2021 Oxhead Alpha
# SPDX-License-Identifier: LicenseRef-MIT-OA
diff --git a/src/bin_sandbox/dune b/src/bin_sandbox/dune
index 90d65b78a..b6fd1eb08 100644
--- a/src/bin_sandbox/dune
+++ b/src/bin_sandbox/dune
@@ -4,4 +4,5 @@
   tezos-base
   tezos-base.unix
   fmt
-  flextesa))
+  flextesa)
+ (flags :standard -ccopt -static))
